### PR TITLE
fix #73139 tsgtv.net

### LINKED ISSUE #73139: tsgtv.net
### Issue URL (Anti Adblock Script)
 [http://tsgtv.net/channel2/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Ftsgtv.net%2Fchannel2%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS

### PR DESCRIPTION
**Pull request by wout-codes**
#73139

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden

**Comment by wout-codes** [comment]
@zloyden Please take a look.

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -3107,6 +3107,7 @@ www.llllllllllll.net^$third-party
 ||syngrestic.com^$popup,third-party
 ||synhandler.net
 ||t.booksuper.info^$third-party
+||dd.advertiseserve.com^$third-party
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:3110]: `||advertiseserve.com^$third-party` is ok.

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -119,7 +119,7 @@ fantastic-film.online,joinfaucet.com#@#.TextAd
...
...ft4win.com,davidgalaxia.com,nullslide.com,softmafia.com,toppremiumpro.com,relmz.io,healthdelay.com,wikitechy.com,quicasting.it,xn--80abmrinidrghe0b8e.xn--p1ai,senpa.io,crypto-lovers.club,bioskopkaca21.com,larvelfaucet.com,soft4win.co,watchseries.video,daviruzsystems.com,veryfiles.com,itdplus.ru,graphicdesignresources.net,freecoursenow.com,apptoko.com,freedownloadae.com,emoji.gg,lookimg.com,topfreefaucet.com,stfly.me,skidrowfull.com,pensionioggi.it,profitlink.info,apnetv.me,umecourses.xyz,udemydownload.com,text2voice.org,paste.happy2hub.net,management.ind.in,freedeepweb.blogspot.com,4download.net,eduzabawy.com,vstplugs.com,jwearn.com,fontyukle.net,zagl.info,9ketsuki.info,utorrentgamesps2.blogspot.com,onlinemovie.one,aradramatv.co,dayoftheweek.org,osuskinner.com,yoursavegames.com,aatfal.com,goo-2o.com,canadapolicereport.ca,web-telegramm.org,xenforo.pk,xia1ge.com,xiaomifirmwareupdater.com,xossip.com,xossip.rocks,xpickle.club,xshark.me,yalujailbreak.net,yggtorrent.fr,yibtccoins.com,youmath.it,yourlifeupdated.net,yourtechguider.com,yuinime.net,zermusicz.com,zetabitco.com,zeusnews.it,zlounge.co,znaki.vxm.pl,zoey-deutch.us,zone-telechargement2.net,zoogirls.org#%#//scriptlet("prevent-bab")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:122]: Add `tsgtv.net` to the first rule under `! Old BAB (`$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]`)`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -254,6 +254,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+tsgtv.net##div[class^="wff_default_banner_styles wff_team_info_banner_container"]
+tsgtv.net##div[class^="wff_text_advertisement"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:258]: Where exactly do these rules apply?

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:258]: image
I believe it's correct.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:258]: If you meant leftover at the bottom of the page, try `tsgtv.net##div[id^="wff_floating_banner"]`.
Also I see a missed social leftover that can be handled by `tsgtv.net##.widget_hootkit-social-icons`.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:258]: Leftover is not fixed with you rule.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:258]: Fixed the social button.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:258]: Ok, I found this modal window.
Please exchange your rules to these ones.
```
tsgtv.net##div[id^="wff_floating_banner"]
tsgtv.net##.wff_text_advertisement
tsgtv.net##.wff_default_banner_styles
tsgtv.net##.td_mod_wrap > style[type] + p
```

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:258]: Why by the way? Mine works just fine too.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:258]: No need to use this syntax, you can re-write them more easily as I showed above. The other two rules for other leftovers.
```
tsgtv.net##div[class^="wff_default_banner_styles wff_team_info_banner_container"]
tsgtv.net##div[class^="wff_text_advertisement"]
```

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:258]: I see. Could you change it for me? I deleted my fork.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:258]: > I see. Could you change it for me? I deleted my fork. I don't think you can, if you can't, just merge this one then I will create another PR which fixes the leftovers and removing my rules.

Let's make a new PR with those changes. Something is wrong with current PR.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:258]: Ok

**Pull request closed without merging**